navigationrail/navigationrail/src/main/java/com/microsoft/device/dualscreen/navigationrail/NavigationRailView.kt

```diff
@@ -199,10 +198,12 @@ class NavigationRailView : NavigationRailView {
 
             normalizeFoldingFeatureRectForView().let { hingeRect ->
                 hingePosition = hingeRect
-                (context as Activity).windowManager.defaultDisplay.getSize(screenSize)
 
-                statusBarHeight = screenSize.y - appWindowFrameHeight
+                val windowBounds = WindowMetricsCalculator.getOrCreate()
+                    .computeCurrentWindowMetrics(context as Activity).bounds
```

Comment:
If this component is added in a Fragment, will this cast make the library to crash?